products/org.eclipse.passage.lbc.server.product/org.eclipse.passage.lbc.server.product

```diff
@@ -82,40 +82,47 @@
       <plugin id="org.eclipse.osgi.compatibility.state" fragment="true"/>
       <plugin id="org.eclipse.osgi.services"/>
       <plugin id="org.eclipse.osgi.util"/>
-      <plugin id="org.eclipse.passage.lic.api"/>
-      <plugin id="org.eclipse.passage.lic.base"/>
-      <plugin id="org.eclipse.passage.lic.equinox"/>
-      <plugin id="org.eclipse.passage.lic.net"/>
       <plugin id="org.eclipse.passage.lbc.api"/>
       <plugin id="org.eclipse.passage.lbc.base"/>
       <plugin id="org.eclipse.passage.lbc.equinox"/>
       <plugin id="org.eclipse.passage.lbc.jetty"/>
       <plugin id="org.eclipse.passage.lbc.server"/>
+      <plugin id="org.eclipse.passage.lic.api"/>
+      <plugin id="org.eclipse.passage.lic.base"/>
+      <plugin id="org.eclipse.passage.lic.equinox"/>
+      <plugin id="org.eclipse.passage.lic.net"/>
+      <plugin id="org.eclipse.pde.ds.lib"/>
       <plugin id="org.sat4j.core"/>
       <plugin id="org.sat4j.pb"/>
+      <plugin id="org.slf4j.api"/>
       <plugin id="org.tukaani.xz"/>
    </plugins>
 
    <features>
-      <feature id="org.eclipse.passage.lbc.feature"/>
       <feature id="org.eclipse.equinox.server.core"/>
       <feature id="org.eclipse.equinox.server.jetty"/>
       <feature id="org.eclipse.equinox.server.p2"/>
-      <feature id="org.eclipse.ecf.filetransfer.httpclient4.feature"/>
       <feature id="org.eclipse.ecf.core.ssl.feature"/>
-      <feature id="org.eclipse.ecf.filetransfer.httpclient4.ssl.feature"/>
       <feature id="org.eclipse.ecf.filetransfer.ssl.feature"/>
       <feature id="org.eclipse.ecf.core.feature"/>
       <feature id="org.eclipse.ecf.filetransfer.feature"/>
+      <feature id="org.eclipse.ecf.filetransfer.httpclient45.feature" installMode="root"/>
+      <feature id="org.eclipse.passage.lbc.feature"/>
```

Comment:
if you declared lbc.execute this feature also should be reworked to avoid confusion